tests/scene/test_theme.h

```diff
@@ -101,18 +101,24 @@ TEST_CASE_FIXTURE(Fixture, "[Theme] Good theme type names") {
 
 	SUBCASE("set_type_variation") {
 		for (const StringName &name : names) {
+			if (name == StringName()) { // Skip empty here, not allowed.
+				return;
+			}
```

Comment:
FYI @YuriSizov. Another option would be to keep the original logic but conditionally disable the error reporting, but it's a bit ugly.

Reply:
I haven't made these, and I definitely don't like how it was written. But shouldn't it be `CHECK_FALSE` then? That's what others do too.

Reply:
Hm indeed, so why did tests pass? xD

Reply:
Ah lol `return` instead of `continue`.